src/MSBuildLocator/build/TraverseRuntimeIdentifiers.targets

```diff
@@ -0,0 +1,31 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Target Name="TraverseRuntimeIdentifiers" AfterTargets="Restore">
+    <PropertyGroup>
+      <RuntimeIdentifiers>
```

Comment:
I'm worried about getting behind as the RID list updates - can this be data-driven in some way? Is there a rule we could follow here?

Reply:
I checked the list of available platform-specific packages and included them
![image](https://github.com/microsoft/MSBuildLocator/assets/95473390/cbb520c1-70d3-48fd-b590-414388c814b8)